<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1011</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="control_panel_groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>421</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : Lavender;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
    <widget class="QLabel" name="control_panel_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight : bold; color : DarkRed; </string>
     </property>
     <property name="text">
      <string>Control Panel</string>
     </property>
    </widget>
    <widget class="QLabel" name="communication_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight : bold; color : Navy;</string>
     </property>
     <property name="text">
      <string>Communication:</string>
     </property>
    </widget>
    <widget class="QLabel" name="mode_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight : bold; color : Navy;</string>
     </property>
     <property name="text">
      <string>Select Node:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="setting_pushButton">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight : bold; background-color : LightSteelBlue; color : DarkSlateGray;</string>
     </property>
     <property name="text">
      <string>Setting</string>
     </property>
     <property name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/imgs/icon_image/port1_rsz.png</normaloff>:/imgs/icon_image/port1_rsz.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="run_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight : bold; background-color : LightSteelBlue; color : DarkSlateGray;</string>
     </property>
     <property name="text">
      <string>Run</string>
     </property>
     <property name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/imgs/icon_image/icon_run.png</normaloff>:/imgs/icon_image/icon_run.png</iconset>
     </property>
    </widget>
    <widget class="QLabel" name="init_velocity_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight : bold; color : Navy;</string>
     </property>
     <property name="text">
      <string>Initial Velocity:</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="velocity_textedit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>170</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Ubuntu';&quot;&gt;40&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Ubuntu';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pause_resume_button">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>210</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight : bold; background-color : LightSteelBlue; color : DarkSlateGray;</string>
     </property>
     <property name="text">
      <string>Pause</string>
     </property>
     <property name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/imgs/icon_image/icon_pause.png</normaloff>:/imgs/icon_image/icon_pause.png</iconset>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>130</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="Resources.qrc">:/imgs/icon_image/icon_info_small.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="connected_label">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>40</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Please set COM port!</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>40</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="Resources.qrc">:/imgs/icon_image/icon_info_small.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="mode_label_3">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>170</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>cm/s</string>
     </property>
    </widget>
    <widget class="QLabel" name="mode_label_4">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>130</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Please select node to run!</string>
     </property>
    </widget>
    <widget class="QPushButton" name="manual_mode_button">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>210</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight : bold; background-color : LightSteelBlue; color : DarkSlateGray;</string>
     </property>
     <property name="text">
      <string>Manual Mode</string>
     </property>
     <property name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/imgs/icon_image/120787.png</normaloff>:/imgs/icon_image/120787.png</iconset>
     </property>
    </widget>
    <widget class="QComboBox" name="nodes_comboBox">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>90</y>
       <width>41</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="nodes_i4_pushButton">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>90</y>
       <width>181</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight : bold; background-color : LightSteelBlue; color : DarkSlateGray;</string>
     </property>
     <property name="text">
      <string>Node Information</string>
     </property>
     <property name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/imgs/icon_image/icon_nodeMap.png</normaloff>:/imgs/icon_image/icon_nodeMap.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="general_status_groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>421</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : Lavender;
</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="g_status_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight : bold; color : DarkRed; </string>
     </property>
     <property name="text">
      <string>Monitor</string>
     </property>
    </widget>
    <widget class="QLabel" name="mode_label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight : bold; color : Navy;</string>
     </property>
     <property name="text">
      <string>Mode:</string>
     </property>
    </widget>
    <widget class="QLabel" name="init_velocity_label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight : bold; color : Navy;</string>
     </property>
     <property name="text">
      <string>Velocity:</string>
     </property>
    </widget>
    <widget class="QLabel" name="state_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight : bold; color : Navy;</string>
     </property>
     <property name="text">
      <string>State:</string>
     </property>
    </widget>
    <widget class="QLabel" name="battery_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight : bold; color : Navy;</string>
     </property>
     <property name="text">
      <string>Battery:</string>
     </property>
    </widget>
    <widget class="QLabel" name="init_velocity_label_6">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>90</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>cm/s</string>
     </property>
    </widget>
    <widget class="QLabel" name="init_velocity_label_7">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>90</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="init_velocity_label_8">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>60</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>None</string>
     </property>
    </widget>
    <widget class="QLabel" name="init_velocity_label_9">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>120</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>150</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>90</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QProgressBar#progressBar
{
      border:none;   /*无边框*/
      background:rgb(211, 215, 207);
      border-radius:5px;
      text-align:center;   /*文本的位置*/
      color: rgb(229, 229, 229);  /*文本颜色*/
}
 
QProgressBar::chunk 
{
      background-color:rgb(115, 210, 22);
      border-radius:4px;
}
</string>
     </property>
     <property name="value">
      <number>70</number>
     </property>
    </widget>
    <widget class="QLabel" name="battery_label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight : bold; color : Navy;</string>
     </property>
     <property name="text">
      <string>Water Capacity:</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar_3">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>180</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>90</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QProgressBar#progressBar
{
      border:none;   /*无边框*/
      background:rgb(211, 215, 207);
      border-radius:5px;
      text-align:center;   /*文本的位置*/
      color: rgb(229, 229, 229);  /*文本颜色*/
}
 
QProgressBar::chunk 
{
      background-color:rgb(115, 210, 22);
      border-radius:4px;
}
</string>
     </property>
     <property name="value">
      <number>70</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="map_groupBox">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>551</width>
      <height>511</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : Lavender;
</string>
    </property>
    <widget class="QLabel" name="navigation_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight : bold; color : DarkRed; </string>
     </property>
     <property name="text">
      <string>Navigation Map</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="graphicsView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>531</width>
       <height>461</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
    </widget>
    <widget class="QLabel" name="p_charging_label">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>470</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight : bold; color : DarkRed; </string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:700;&quot;&gt;PARKING/CHARGING&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="robot_label">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>440</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight : bold; color : Navy; background-color : White; </string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:700;&quot;&gt;ROBOT&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
